functions/go/export-terraform/go.mod

```diff
@@ -2,11 +2,11 @@ module github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/export
 
 go 1.16
 
-replace sigs.k8s.io/kustomize/kyaml v0.11.1 => github.com/mengqiy/kustomize/kyaml v0.15.0
+replace github.com/GoogleContainerTools/kpt-functions-catalog/thirdparty/kyaml/fnsdk v0.0.0-20220106201853-53d6f3d583fe => ../../../thirdparty/kyaml/fnsdk
```

Comment:
We got some push back when trying to merge the new SDK in upstream. We are actively working on it to address the concerns.
Meanwhile we want to unblock the OC project. So I have moved the code (github.com/mengqiy/kustomize/kyaml v0.15.0) from my fork to the catalog repo in https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/724.